src/core/packages/chrome/browser-internal/src/chrome_service.tsx

```diff
@@ -94,14 +96,16 @@ export interface StartDeps {
   docLinks: DocLinksStart;
   http: InternalHttpStart;
   injectedMetadata: InternalInjectedMetadataStart;
-  notifications: NotificationsStart;
   customBranding: CustomBrandingStart;
   i18n: I18nStart;
   theme: ThemeServiceStart;
   userProfile: UserProfileService;
   uiSettings: IUiSettingsClient;
   analytics: AnalyticsServiceStart;
   featureFlags: FeatureFlagsStart;
+
+  // working around circular dependency
+  notifications: () => Promise<NotificationsStart>;
```

Comment:
we want to pass chrome into rendering service so that we could wrap react roots with chrome provider. for this we need to break circular dep: 

chrome -> rendering -> notifications -> chrome.


This is a bit clunky now and also breaks bunch of unit tests that I haven't fixed yet, but I want to extract this change into a separate PR